src/Estimators/tests/test_WalkerLogCollector.cpp

```diff
@@ -0,0 +1,136 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2024 QMCPACK developers.
+//
+// File developed by: Jaron T. Krogel, krogeljt@ornl.gov, Oak Ridge National Laboratory
+//
+// File created by: Jaron T. Krogel, krogeljt@ornl.gov, Oak Ridge National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#include "catch.hpp"
+
+#include "Message/Communicate.h"
+#include "QMCHamiltonians/QMCHamiltonian.h"
+#include "Particle/tests/MinimalParticlePool.h"
+#include "QMCHamiltonians/tests/MinimalHamiltonianPool.h"
+#include "QMCWaveFunctions/tests/MinimalWaveFunctionPool.h"
+#include "Utilities/ResourceCollection.h"
+#include "Utilities/ProjectData.h"
+
+#include "Estimators/WalkerLogCollector.h"
+
+
+namespace qmcplusplus
+{
+
+TEST_CASE("WalkerLogCollector::collect", "[estimators]")
+{
+  std::cout <<"\n\n=======================================================\n";
```

Comment:
Please replace all the `std::cout` with `app_log()` so we can suppress printout by default.

Reply:
Done